### ISSUE TITLE
Suggestion - Add an option to change the icon colors

### ISSUE THREAD

**Issue by AntogamerYT** [ByPikod/markdown-it-github-alert#1]
Hello, I'm using your plugin and it seems pretty cool so far, but i had an issue:
Due to how your plugin works, the icon of the alerts always seems to be black and it doesn't really fit my page:
![image](https://github.com/ByPikod/markdown-it-github-alert/assets/64664639/f980fae3-d4e7-4370-b14c-4680a45c957b)

I wanted to ask if you could add a way to easily modify the icon's colors by maybe adding a config passable to the plugin via the `use` function of markdown-it (i think it's doable since you're using svgs)
A little example from matkdown-it-anchor in case you didn't get my explanation:
```js
markdownIt({ html: true }).use(
    markdownItAnchor,
    {
      permalink: true,
      permalinkClass: 'anchor',
      permalinkSymbol: '#'
    }
  )
  ```
  
I also noticed that not all warnings from GitHub are present in your lib, github currently supports

> [!NOTE]  
> Notes (supported by your plugin)

> [!TIP]
> Tip (unsupported)

> [!IMPORTANT]  
> Important (supported)

> [!WARNING]  
> Warning (supported)

> [!CAUTION]
> Caution (unsupported)

**Comment by ByPikod** [comment]
Hello, thanks for you feedback!

### Unsupported Types of Alerts
I didn't know there is more types of alerts. I am going to implement them as soon as possible.

### Uncolored Icon
And about the issue of symbols being uncolored, this plugin of Markdown-It has been originally created to provide a preview for markdown files in Visual Studio Code. And after looking into other plugins as an example, I couldn't find one that provides built-in CSS. So I added my CSS into VS Code extension. Here is the link for it:
https://github.com/ByPikod/vscode-markdown-alert/

But your feedback reminded me that people might want the following CSS file in their project. 
https://github.com/ByPikod/vscode-markdown-alert/blob/main/style.css

I am going to add this CSS file into this repository as well. And If you wonder how it looks like with the CSS on it here is a preview:

![preview](https://github.com/ByPikod/markdown-it-github-alert/assets/46069238/0f7c7961-b3e2-4b50-b9c2-55b966bff86e)

Long story short, if you want to change the color of the icons, you can use "fill" property of CSS. So I don't think we need a configuration for that. But if you think we still need this configuration, please let me know with a more clear explanation of why.

**Comment by ByPikod** [comment]
All types of alert boxes are now supported. Here is preview for it:

![preview](https://github.com/ByPikod/markdown-it-github-alert/assets/46069238/60ecd4f4-a413-4ce9-b64c-19b3c0dc5947)

I am closing the issue for now but as I said before:
>  ... if you think we still need this configuration, please let me know with a more clear explanation of why.

Do not hesitate opening this issue back if you think so.
